concrete-core-fixture/src/generation/prototyping/container.rs

```diff
@@ -0,0 +1,37 @@
+use crate::generation::prototypes::{ContainerPrototype, ProtoVec32, ProtoVec64};
+// , ProtoMutSlice32, ProtoMutSlice64, ProtoSlice32, ProtoSlice64
+use crate::generation::{IntegerPrecision, Maker, Precision32, Precision64};
+
+/// A trait allowing to manipulate container prototypes.
+pub trait PrototypesContainer<Precision: IntegerPrecision> {
+    type ContainerProto: ContainerPrototype<Precision = Precision>;
+    fn transform_raw_to_container(&mut self, raw: &[Precision::Raw]) -> Self::ContainerProto;
+    fn transform_container_to_raw(
```

Comment:
Here, to be consistent with other `Prototypes*` trait, we should use `raw_vec` instead of `raw`.

Reply:
will do!

Reply:
done